### ISSUE TITLE
mmo add service name fails with 'no such file or directory'

### ISSUE THREAD

**Issue by rysavyvladan** [flowup/mmo#58]
if you are trying generate new service, you get this error
```
DEBU[0000] Bootstrap successful                         
[info]
FATA[0000] open transaction/access.go: no such file or directory 
```

**Comment by rysavyvladan** [comment]
it is looking thet this error is fixed by https://github.com/flowup/mmo/tree/feature/refactor

**Comment by kubaj** [comment]
Fixed in #56
